core/controllers/editor_test.py

```diff
@@ -2591,6 +2591,8 @@ def test_delete_learner_answer_info_of_exploration_states(self):
                     self.state_name), expected_status_int=404)
 
     def test_delete_learner_answer_info_of_question_states(self):
+        self.logout()
```

Comment:
Hm. Probably cleaner to have the self.logout() at the end of setUp(), and add a self.login() at the start of other tests?

Reply:
Done